### PR TITLE
Update docs to reflect post-philly policies

### PR DESCRIPTION
**Pull request by titaniumbones**
- no longer encourage forking of tool
- explain new subprimer system

@trinberg @dcwalk if it looks good please merge!

### CODE REVIEW

**Comment by titaniumbones** [comment]
If someone else has time to review and pull that would be great...

File: Contributing.md [hunk 1]
```diff
@@ -13,9 +38,8 @@ You can use this tool as currently designed for your own event. At present, it p
 
 2. For each field name, you will have to identify the corresponding field entry id and update the various field-relate variables. In your browser's developer tools pane, `inspect element` on the input field. You should see `entry.[integer]` in the highlighted code. That is the value for the entry field ID. 
 
-3. The first time a user nominates a seed, they will choose a government agency. Subsequent nominations will default to that same agencies, but the default list here is extensive. If you are modifying the extension anyway, you may want to filter out this list in the `#agency` field of `popup.html`.
+3. The first time a user nominates a seed, they will choose a government agency subagency, organization, and possibly suborganization. Subsequent nominations will default to that same values. If you are modifying the extension anyway, you may want to filter out this list in the `#agency` field of `popup.html`.
```
Comment (dcwalk) [inline Contributing.md]: May want to remove this last line now?

File: Contributing.md [hunk 2]
```diff
@@ -13,9 +38,8 @@ You can use this tool as currently designed for your own event. At present, it p
...
 
-3. The first time a user nominates a seed, they will choose a government agency. Subsequent nominations will default to that same agencies, but the default list here is extensive. If you are modifying the extension anyway, you may want to filter out this list in the `#agency` field of `popup.html`.
+3. The first time a user nominates a seed, they will choose a government agency subagency, organization, and possibly suborganization. Subsequent nominations will default to that same values. If you are modifying the extension anyway, you may want to filter out this list in the `#agency` field of `popup.html`. 
 
-Small events can uncomment the "notificationToolUrl"-related lines -- this will permit users to add seeds directly to the nomination tool database. However, there is some worry that large numbers of users could crash the nomination tool server; so for large events, we've commented this out.
+4. If you're modifying this extension for  non-environment-related event, you will probably want to imitate or extend the EDGi agency taonomy.  Contact us for more information, and/or read our [agency primers](https://envirodatagov.org/agency-forecasts/), which you will want to modify.
```
Comment (dcwalk) [inline Contributing.md]: two spaces

taxonomy*

**Review by dcwalk — COMMENTED** [review]
Looks good, just a couple small bits

File: Contributing.md (hunk 1 above)
Comment (titaniumbones) [reply Contributing.md]: Sure, yes, thx

**Comment by dcwalk** [comment]
Looks good, merging :)

**Pull request merged**